src/test/java/core/basesyntax/service/impl/FruitShopFileWriterImplTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import org.junit.Test;
+
+public class FruitShopFileWriterImplTest {
+    private final FruitShopFileWriterImpl fruitShopFileWriter = new FruitShopFileWriterImpl();
+    private final String fileName = "./src/test/resources/testFruitsReport.csv";
+    
+    @Test
+    public void write_validReport_ok() {
+        String expected = "123,abc,qwe";
+        String actual = "";
+        fruitShopFileWriter.write(expected, fileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            actual = reader.readLine();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file", e);
+        }
+        assertEquals(expected, actual);
+    }
+}

```

Comment:
1. If the file contains two ore more lines?
2. I think u need to test situation when Path/File will be incorrect.